ggshield/cmd/iac/scan/all.py

```diff
@@ -5,22 +5,17 @@
 from pygitguardian.iac_models import IaCScanParameters, IaCScanResult
 
 from ggshield.cmd.iac.scan.iac_scan_common_options import (
-    add_iac_scan_common_options,
-    update_context,
-)
-from ggshield.cmd.iac.scan.iac_scan_utils import (
-    IaCSkipScanResult,
-    create_output_handler,
-    handle_scan_error,
-)
+    add_iac_scan_common_options, update_context)
+from ggshield.cmd.iac.scan.iac_scan_utils import (IaCSkipScanResult,
+                                                  create_output_handler,
+                                                  handle_scan_error)
```

Comment:
Same here, you should run black on this file I think.

Reply:
This is isort doing wonders :'(